hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/config/EvmConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.hedera.mirror.web3.evm.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EntityScan("com.hedera.mirror.web3.evm")
+@EnableCaching
+public class EvmConfiguration {
+
+    @Bean
+    public Caffeine caffeineConfig() {
```

Comment:
Use package access for class and methods in configuration classes as is mirror node standard.
```suggestion
   Caffeine caffeineConfig() {
```

Reply:
I meant for every method and the class itself, not just the one above.

Reply:
We can't make the class having package access since we refer to the `CACHE_MANAGER_10MIN` and `CACHE_MANAGER_500MS` constants in the `RecordFileRepository`